packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/getuserinfo.md

```diff
@@ -1,42 +1,10 @@
-To find information about the user, use the `getUserProfile()` method. The `UserInfo` object passed into the `onSuccess()` method contains information for the signed-in user.
+To find information about the user, use the `idToken` method. The `Jwt` object contains information for the user.
```

Comment:
This is missing the bridge between the shared text and the code snippets. In the iOS version of this I start with the What (Display the user info), then the how (by updating an empty function they've already entered). I then show the code. After the code I explain what it does.

Consider a similar flow. Lead with what the developer is adding to the app, printing the preferred username. Then have them add the code or perhaps explain how the Kotlin SDK deal with the ID Token. And after that, explain how it gets the local ID token from... etc.

The text that precedes the content of this file is:

> ## Get info about the user
> 
> The ID token returned by Okta contains user information, or *claims*, that are based on the scopes requested by the app (see [Configure your app](#configure-your-app)).
> 
> This app includes the `profile` scope that includes the user's email address, name, and preferred username. You can use this information to update your UI, such as showing the customer's name.
> 
> Use the Okta user information endpoints for items that aren't available in the ID token. For general information on requesting user info, see the [userinfo response example](/docs/reference/api/oidc/#response-example-success-6)).
> 
> <StackSnippet snippet="getuserinfo" />
>